### PR TITLE
Implemented ResultButtonContainer and the functionality

### LINKED ISSUE #228: Frontend : Result Button
**User story**

**Who**: As a User
**What:** I want to be able to click the **Result Button**  which exists on the Board Page when the deadline has passed.
**Why:** so that I can see the final candidates as a result.

**Acceptance criteria**

This is the Result Button after when the deadline has passed.
ResultButton

### PR DESCRIPTION
**Pull request by Saidemm**
# Description

 Result button can be clicked after the deadline is past and will link and pass the boardId to the Result Page.

Fixes #228  (issue)

# How to test?

By clicking the result button when deadline is passed.

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by senner007 — CHANGES_REQUESTED** [review]
Hi 
It has some unrelated changes

File: src/client/containers/BoardPage/ResultButton/ResultButtonPostDeadline.container.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { Link } from 'react-router-dom';
+import GenericButton from '../../../components/GenericButton/GenericButton.component';
+
+const onClick = () => {
+  // console.log('clicked');
+};
+export default function ResultButtonPostDeadline() {
+  const { boardInfo } = useBoard();
+  const boardId = boardInfo.id;
+  console.log('BB', boardId);
```
Comment (shanhaider23) [inline src/client/containers/BoardPage/ResultButton/ResultButtonPostDeadline.container.js]: I think forgot to comment out console.log

**Comment by senner007** [comment]
> Hi
> It has some unrelated changes

Ah, right you want to merge it into the board branch

**Review by senner007 — APPROVED** [review]
👍

**Review by shanhaider23 — APPROVED** [review]

**Pull request merged**